serverlessworkflow/modules/ROOT/pages/cloud/operator/using-persistence.adoc

```diff
@@ -1,26 +1,33 @@
-= Using persistence in the SonataFlow Workflow CR
+= Using persistence in {product_name} workflows
 :compat-mode!:
 // Metadata:
 :description: Using persistence in the workflow instance to store its context
 :keywords: sonataflow, workflow, serverless, operator, kubernetes, persistence
 
+This document describes how to configure a SonataFlow instance to use persistence and store the workflow context in a relational database.
 
-This document describes how to configure a SonataFlow instance to use persistence to store the flow's context in a relational database.
+Kubernetes's pods are stateless by definition. In some scenarios, this can be a challenge for workloads that require maintaining the status of 
+the application regardless of the pod's lifecycle. In the case of {product_name}, by default, the context of the workflow is lost when the pod restarts.
 
-== Configuring the SonataFlow CR to use persistence
+If your workflow requires recovery from such scenarios, you must provide additional configuration to enable the xref:persistence/core-concepts.adoc#_workflow_runtime_persistence[Workflow Runtime Persistence].
+That configuration must be provided by using the <<_configuring_the_persistence_using_the_sonataflowplatform_cr, SonataFlowPlatform CR>> or the <<_configuring_the_persistence_using_the_sonataflow_cr, SonataFlow CR>>, and has different scopes depending on each case.
 
-Kubernetes's pods are stateless by definition. In some scenarios, this can be a challenge for workloads that require maintaining the status of 
-the application regardless of the pod's lifecycle. In the case of {product_name}, the context of the workflow is lost when the pod restarts.
-If your workflow requires recovery from such scenarios, you have to make these additions to your workflow CR:
-Use the `persistence` field in the `SonataFlow` workflow spec to define the database service located in the same cluster.
-There are 2 ways to accomplish this:
+== Configuring the persistence using the SonataFlowPlatform CR
 
-* Using the Platform CR's defined persistence
-When the Platform CR is deployed with its persistence spec populated it enables workflows to leverage its configuration to populate the persistence
-properties in the workflows. 
+The SonataFlowPlatform CR facilitates the configuration of the persistence with namespace scope. It means that it will be automatically applied to all the workflows deployed in
```

Comment:
```suggestion
The `SonataFlowPlatform` CR facilitates the configuration of the persistence with namespace scope. It means that it will be automatically applied to all the workflows deployed in
```